specification/network/resource-manager/Microsoft.Network/stable/2020-08-01/privateLinkService.json

```diff
@@ -769,6 +769,10 @@
   "definitions": {
     "PrivateLinkService": {
       "properties": {
+        "extendedLocation": {
+          "$ref": "./network.json#/definitions/ExtendedLocation",
+          "description": "The extended location of the load balancer."
+        },
```

Comment:
Is extentendedLocation a replacement of edgeZone?

Reply:
Can you use extentendedLocation in examples?

Reply:
I am removing the edgeZone property to revert a change that I recently made.

I added examples.